vanilla-framework/vanilla-framework
# New example: Responsive logo strip using the proposed u-first-col--* classes

## Done

This is meant to replace the inline images pattern. In order to reduce the amount of rewriting necessary to move a site to vanilla 2.0, I'd suggest deprecating the inline images pattern on a point release rather than 2.0.

Also unlike inline-images, this doesn't need to be a pattern - it can be built with vanilla plain grid classes. Still useful to have as a reference implementation we can point people to.

## QA

- Pull code
- Run `./run serve --watch`
- Open http://0.0.0.0:8101/vanilla-framework/logo-strip.html
- Verify logo strip looks ok on all breakpoints.

## Details

[List of links to issues/bugs and cards if needed]

## Screenshots

[if relevant, include a screenshot]
